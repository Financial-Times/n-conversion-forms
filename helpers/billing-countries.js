import { billingCountries } from 'n-common-static-data';

export { billingCountries };
