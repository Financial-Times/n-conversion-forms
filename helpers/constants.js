const printRegions = {
	cemeaV1: 'CEMEA_V1',
	cemeaV2: 'CEMEA_V2',
	apac: 'APAC',
};

const addressLine1AutoComplete = 'address-line1';
const addressLine2AutoComplete = 'address-line2';
const addressLine3AutoComplete = 'address-line3';

module.exports = {
	printRegions,
	addressLine1AutoComplete,
	addressLine2AutoComplete,
	addressLine3AutoComplete,
};
