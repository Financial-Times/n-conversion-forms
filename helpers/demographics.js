const { demographics } = require('n-common-static-data');

export { demographics };
