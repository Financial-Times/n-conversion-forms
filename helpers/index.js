module.exports = {
	'ncf-countries': require('./ncf-countries'),
	'ncf-common-data': require('./ncf-common-data'),
	cemeaV1ISO: require('./supportedCountries').cemeaV1ISO,
	cemeaV2ISO: require('./supportedCountries').cemeaV2ISO,
	apacISO: require('./supportedCountries').apacISO,
	identifyFTShippingZone: require('./supportedCountries')
		.identifyFTShippingZone,
	countriesSupported: require('./supportedCountries').countriesSupported,
	countriesSupportedISO: require('./supportedCountries').countriesSupportedISO,
	supportedPostcodeValidators: require('./supportedPostcodeValidators')
		.supportedPostcodeValidators,
	allSupportedPostcodeExamples: require('./supportedPostcodeExamples')
		.allSupportedPostcodeExamples,
	demographics: require('./demographics').demographics,
	billingCountries: require('./billing-countries').billingCountries,
};
