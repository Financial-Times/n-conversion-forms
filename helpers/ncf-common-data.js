const _get = require('lodash.get');
const COMMON_DATA = require('n-common-static-data');

module.exports = function ({ hash, fn }) {
	const context = Object.assign({}, { [hash.export]: _get(COMMON_DATA, hash.import) }, this);
	return fn(context);
};
