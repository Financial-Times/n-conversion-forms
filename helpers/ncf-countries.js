const { countries } = require('n-common-static-data').billingCountries;

const MINIMUM_TO_GROUP = 50;
const MINIMUM_TO_SHOW_GROUPS = 2;

module.exports = function ({ hash = {}, fn }) {
	const filter = Array.isArray(hash.filterList) ? hash.filterList : false;
	const value = hash.value;

	let data = countries;

	// Only show countries in the filter
	if (filter) {
		data = data.filter(item => filter.includes(item.code));
	}

	// Select the country
	if (value) {
		data = data.map(item => {
			item.selected = item.code === value;
			return item;
		});
	}

	// Group countries
	if (data.length >= MINIMUM_TO_GROUP) {
		data = groupCountries(data);
	}

	const context = Object.assign({ countries: data }, this);
	return fn(context);
};

/**
 * Produce a frequently used option group
 * @param {Array} countries
 * @return {Array}
 */
function groupCountries (countries) {
	const frequentlyUsed = ['GBR', 'USA', 'JPN', 'FRA', 'CAN'];

	const frequentlyUsedCountries = countries
		.filter(item => frequentlyUsed.includes(item.code))
		.map(item => Object.assign({}, item))
		.sort((a, b) => frequentlyUsed.indexOf(a.code) - frequentlyUsed.indexOf(b.code));

	// Not worth showing return standard countries list
	if (frequentlyUsedCountries.length < MINIMUM_TO_SHOW_GROUPS) {
		return countries;
	}

	// If selected country is in frequently used only leave that one selected
	// otherwise the user is shown the lowest selected option
	const selected = frequentlyUsedCountries.find(item => item.selected);
	if (selected) {
		countries.forEach(item => item.selected = false);
	}

	return [
		{ label: 'Frequently Used', countries: frequentlyUsedCountries },
		{ label: 'Alphabetical', countries }
	];
}
