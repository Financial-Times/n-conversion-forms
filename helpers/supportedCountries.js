const supportedCountriesMasterList = {
	cemeaV1: {
		AUT: 'Austria',
		BEL: 'Belgium',
		BGR: 'Bulgaria',
		HRV: 'Croatia',
		CZE: 'Czechia',
		DNK: 'Denmark',
		FIN: 'Finland',
		DEU: 'Germany',
		GRC: 'Greece',
		HUN: 'Hungary',
		ISL: 'Iceland',
		ITA: 'Italy',
		LIE: 'Liechtenstein',
		LTU: 'Lithuania',
		NLD: 'Netherlands',
		NOR: 'Norway',
		POL: 'Poland',
		PRT: 'Portugal',
		QAT: 'Qatar',
		ROU: 'Romania',
		RUS: 'Russian Federation',
		SVK: 'Slovakia',
		SVN: 'Slovenia',
		ZAF: 'South Africa',
		ESP: 'Spain',
		SWE: 'Sweden',
		CHE: 'Switzerland'
	},
	cemeaV2: { FRA: 'France', LUX: 'Luxembourg', MCO: 'Monaco' },
	apac: {
		CHN: 'China',
		HKG: 'Hong Kong',
		IND: 'India',
		IDN: 'Indonesia',
		JPN: 'Japan',
		KOR: 'Korea, Republic of',
		MYS: 'Malaysia',
		PHL: 'Philippines',
		SGP: 'Singapore',
		TWN: 'Taiwan, Province of China',
		THA: 'Thailand'
	},
	GBR: 'United Kingdom',
	USA: 'United States',
	CAN: 'Canada'
};


const cemeaV1ISO = Object.keys(supportedCountriesMasterList.cemeaV1);
const cemeaV2ISO = Object.keys(supportedCountriesMasterList.cemeaV2);
const apacISO = Object.keys(supportedCountriesMasterList.apac);
const countriesSupported = Object.assign({}, ...(function _flatten (o) { return [].concat(...Object.keys(o).map(k => typeof o[k] === 'object' ? _flatten(o[k]) : ({[k]: o[k]})));}(supportedCountriesMasterList)));


module.exports = { cemeaV1ISO, cemeaV2ISO, apacISO, countriesSupported };
