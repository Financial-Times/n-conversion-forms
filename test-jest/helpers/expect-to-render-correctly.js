import ReactDOMServer from 'react-dom/server';

export const expectToRenderCorrectly = {
	toRenderCorrectly(reactComponent, props) {
		// Render the React component with props.
		const ReactComponentRendered = ReactDOMServer.renderToStaticMarkup(
			reactComponent(props)
		);

		// eslint-disable-next-line jest/no-standalone-expect
		expect(ReactComponentRendered).toMatchSnapshot();

		// This point is reached when the above assertions were successful.
		// The test should therefore always pass, that means it needs to be
		// `true` when used normally, and `false` when `.not` was used.
		return {
			pass: !this.isNot,
		};
	},
};
