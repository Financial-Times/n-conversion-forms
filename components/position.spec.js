import { Position } from './index';
import { expectToRenderCorrectly } from '../test-jest/helpers/expect-to-render-correctly';
import { demographics } from 'n-common-static-data';
const defaultOptions = demographics.positions.positions;

import Enzyme, { mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

expect.extend(expectToRenderCorrectly);

describe('Position', () => {
	it('render a select with a label', () => {
		const props = {
			options: defaultOptions
		};

		expect(Position).toRenderCorrectly(props);
	});

	it('can render an initial selected value', () => {
		const props = {
			options: defaultOptions,
			value: 'CP',
		};

		expect(Position).toRenderCorrectly(props);
	});

	it('can render a disable select', () => {
		const props = {
			options: defaultOptions,
			isDisabled: true
		};

		expect(Position).toRenderCorrectly(props);
	});

	it('can render an error message', () => {
		const props = {
			options: defaultOptions,
			hasError: true,
		};

		expect(Position).toRenderCorrectly(props);
	});

	it('can override ids and name for field and select', () => {
		const props = {
			fieldId: 'fieldID',
			selectId: 'selectId',
			selectName: 'selectName',
		};
		const component = mount(Position(props));

		const field = component.find('#fieldID');
		expect(field.exists()).toBe(true);

		const select = component.find('select#selectId');
		expect(select.exists()).toBe(true);
	});
});
