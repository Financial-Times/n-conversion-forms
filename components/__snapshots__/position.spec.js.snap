// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Position can render a disable select 1`] = `
<div id="positionField"
     class="o-forms o-forms--wide ncf__field js-field"
     data-ui-item="select"
     data-ui-item-name="position"
     data-validate="required"
>
  <label for="position"
         class="o-forms__label"
  >
    What&#x27;s your job position?
  </label>
  <select id="position"
          name="position"
          class="o-forms__select js-field__input js-item__value"
          data-trackable="field-position"
          aria-required="true"
          required
          disabled
  >
    <option value>
      Please select a job position
    </option>
    <option value="AN">
      Analyst
    </option>
    <option value="AS">
      Associate
    </option>
    <option value="BR">
      Broker/Trader/Advisor
    </option>
    <option value="BU">
      Business School Academic
    </option>
    <option value="CP">
      CEO/president/Chairman
    </option>
    <option value="CS">
      Consultant
    </option>
    <option value="DT">
      Diplomat/Sen Govt Officer
    </option>
    <option value="OE">
      Exec Mgmt (EVP/SVP/MD)
    </option>
    <option value="GI">
      Gov&#x27;t/Int&#x27;l org official
    </option>
    <option value="MB">
      MBA Student
    </option>
    <option value="MA">
      Manager/Supervisor
    </option>
    <option value="OT">
      Other
    </option>
    <option value="MD">
      Other C Level (CFO/COO/CIO/CMO)
    </option>
    <option value="OP">
      Owner/Partner/Proprietor
    </option>
    <option value="PO">
      Politician/Government Minister
    </option>
    <option value="PR">
      Professional
    </option>
    <option value="PM">
      Programme/Project Manager
    </option>
    <option value="SE">
      Secretary/Treasurer
    </option>
    <option value="SM">
      Senior Manager/Dept Head
    </option>
    <option value="TS">
      Technical/Business Specialist
    </option>
    <option value="DH">
      VP/Director
    </option>
  </select>
  <div class="o-forms__errortext">
    Please select your position
  </div>
</div>
`;

exports[`Position can render a disable select 2`] = `
<div id="positionField"
     class="o-forms o-forms--wide ncf__field js-field"
     data-ui-item="select"
     data-ui-item-name="position"
     data-validate="required"
>
  <label for="position"
         class="o-forms__label"
  >
    What&#x27;s your job position?
  </label>
  <select id="position"
          name="position"
          class="o-forms__select js-field__input js-item__value"
          data-trackable="field-position"
          aria-required="true"
          required
          disabled
  >
    <option value>
      Please select a job position
    </option>
    <option value="AN">
      Analyst
    </option>
    <option value="AS">
      Associate
    </option>
    <option value="BR">
      Broker/Trader/Advisor
    </option>
    <option value="BU">
      Business School Academic
    </option>
    <option value="CP">
      CEO/president/Chairman
    </option>
    <option value="CS">
      Consultant
    </option>
    <option value="DT">
      Diplomat/Sen Govt Officer
    </option>
    <option value="OE">
      Exec Mgmt (EVP/SVP/MD)
    </option>
    <option value="GI">
      Gov&#x27;t/Int&#x27;l org official
    </option>
    <option value="MB">
      MBA Student
    </option>
    <option value="MA">
      Manager/Supervisor
    </option>
    <option value="OT">
      Other
    </option>
    <option value="MD">
      Other C Level (CFO/COO/CIO/CMO)
    </option>
    <option value="OP">
      Owner/Partner/Proprietor
    </option>
    <option value="PO">
      Politician/Government Minister
    </option>
    <option value="PR">
      Professional
    </option>
    <option value="PM">
      Programme/Project Manager
    </option>
    <option value="SE">
      Secretary/Treasurer
    </option>
    <option value="SM">
      Senior Manager/Dept Head
    </option>
    <option value="TS">
      Technical/Business Specialist
    </option>
    <option value="DH">
      VP/Director
    </option>
  </select>
  <div class="o-forms__errortext">
    Please select your position
  </div>
</div>
`;

exports[`Position can render an error message 1`] = `
<div id="positionField"
     class="o-forms o-forms--wide ncf__field js-field o-forms--error"
     data-ui-item="select"
     data-ui-item-name="position"
     data-validate="required"
>
  <label for="position"
         class="o-forms__label"
  >
    What&#x27;s your job position?
  </label>
  <select id="position"
          name="position"
          class="o-forms__select js-field__input js-item__value"
          data-trackable="field-position"
          aria-required="true"
          required
  >
    <option value>
      Please select a job position
    </option>
    <option value="AN">
      Analyst
    </option>
    <option value="AS">
      Associate
    </option>
    <option value="BR">
      Broker/Trader/Advisor
    </option>
    <option value="BU">
      Business School Academic
    </option>
    <option value="CP">
      CEO/president/Chairman
    </option>
    <option value="CS">
      Consultant
    </option>
    <option value="DT">
      Diplomat/Sen Govt Officer
    </option>
    <option value="OE">
      Exec Mgmt (EVP/SVP/MD)
    </option>
    <option value="GI">
      Gov&#x27;t/Int&#x27;l org official
    </option>
    <option value="MB">
      MBA Student
    </option>
    <option value="MA">
      Manager/Supervisor
    </option>
    <option value="OT">
      Other
    </option>
    <option value="MD">
      Other C Level (CFO/COO/CIO/CMO)
    </option>
    <option value="OP">
      Owner/Partner/Proprietor
    </option>
    <option value="PO">
      Politician/Government Minister
    </option>
    <option value="PR">
      Professional
    </option>
    <option value="PM">
      Programme/Project Manager
    </option>
    <option value="SE">
      Secretary/Treasurer
    </option>
    <option value="SM">
      Senior Manager/Dept Head
    </option>
    <option value="TS">
      Technical/Business Specialist
    </option>
    <option value="DH">
      VP/Director
    </option>
  </select>
  <div class="o-forms__errortext">
    Please select your position
  </div>
</div>
`;

exports[`Position can render an error message 2`] = `
<div id="positionField"
     class="o-forms o-forms--wide ncf__field js-field o-forms--error"
     data-ui-item="select"
     data-ui-item-name="position"
     data-validate="required"
>
  <label for="position"
         class="o-forms__label"
  >
    What&#x27;s your job position?
  </label>
  <select id="position"
          name="position"
          class="o-forms__select js-field__input js-item__value"
          data-trackable="field-position"
          aria-required="true"
          required
  >
    <option value>
      Please select a job position
    </option>
    <option value="AN">
      Analyst
    </option>
    <option value="AS">
      Associate
    </option>
    <option value="BR">
      Broker/Trader/Advisor
    </option>
    <option value="BU">
      Business School Academic
    </option>
    <option value="CP">
      CEO/president/Chairman
    </option>
    <option value="CS">
      Consultant
    </option>
    <option value="DT">
      Diplomat/Sen Govt Officer
    </option>
    <option value="OE">
      Exec Mgmt (EVP/SVP/MD)
    </option>
    <option value="GI">
      Gov&#x27;t/Int&#x27;l org official
    </option>
    <option value="MB">
      MBA Student
    </option>
    <option value="MA">
      Manager/Supervisor
    </option>
    <option value="OT">
      Other
    </option>
    <option value="MD">
      Other C Level (CFO/COO/CIO/CMO)
    </option>
    <option value="OP">
      Owner/Partner/Proprietor
    </option>
    <option value="PO">
      Politician/Government Minister
    </option>
    <option value="PR">
      Professional
    </option>
    <option value="PM">
      Programme/Project Manager
    </option>
    <option value="SE">
      Secretary/Treasurer
    </option>
    <option value="SM">
      Senior Manager/Dept Head
    </option>
    <option value="TS">
      Technical/Business Specialist
    </option>
    <option value="DH">
      VP/Director
    </option>
  </select>
  <div class="o-forms__errortext">
    Please select your position
  </div>
</div>
`;

exports[`Position can render an initial selected value 1`] = `
<div id="positionField"
     class="o-forms o-forms--wide ncf__field js-field"
     data-ui-item="select"
     data-ui-item-name="position"
     data-validate="required"
>
  <label for="position"
         class="o-forms__label"
  >
    What&#x27;s your job position?
  </label>
  <select id="position"
          name="position"
          class="o-forms__select js-field__input js-item__value"
          data-trackable="field-position"
          aria-required="true"
          required
  >
    <option value>
      Please select a job position
    </option>
    <option value="AN">
      Analyst
    </option>
    <option value="AS">
      Associate
    </option>
    <option value="BR">
      Broker/Trader/Advisor
    </option>
    <option value="BU">
      Business School Academic
    </option>
    <option selected
            value="CP"
    >
      CEO/president/Chairman
    </option>
    <option value="CS">
      Consultant
    </option>
    <option value="DT">
      Diplomat/Sen Govt Officer
    </option>
    <option value="OE">
      Exec Mgmt (EVP/SVP/MD)
    </option>
    <option value="GI">
      Gov&#x27;t/Int&#x27;l org official
    </option>
    <option value="MB">
      MBA Student
    </option>
    <option value="MA">
      Manager/Supervisor
    </option>
    <option value="OT">
      Other
    </option>
    <option value="MD">
      Other C Level (CFO/COO/CIO/CMO)
    </option>
    <option value="OP">
      Owner/Partner/Proprietor
    </option>
    <option value="PO">
      Politician/Government Minister
    </option>
    <option value="PR">
      Professional
    </option>
    <option value="PM">
      Programme/Project Manager
    </option>
    <option value="SE">
      Secretary/Treasurer
    </option>
    <option value="SM">
      Senior Manager/Dept Head
    </option>
    <option value="TS">
      Technical/Business Specialist
    </option>
    <option value="DH">
      VP/Director
    </option>
  </select>
  <div class="o-forms__errortext">
    Please select your position
  </div>
</div>
`;

exports[`Position can render an initial selected value 2`] = `
<div id="positionField"
     class="o-forms o-forms--wide ncf__field js-field"
     data-ui-item="select"
     data-ui-item-name="position"
     data-validate="required"
>
  <label for="position"
         class="o-forms__label"
  >
    What&#x27;s your job position?
  </label>
  <select id="position"
          name="position"
          class="o-forms__select js-field__input js-item__value"
          data-trackable="field-position"
          aria-required="true"
          required
  >
    <option value>
      Please select a job position
    </option>
    <option value="AN">
      Analyst
    </option>
    <option value="AS">
      Associate
    </option>
    <option value="BR">
      Broker/Trader/Advisor
    </option>
    <option value="BU">
      Business School Academic
    </option>
    <option selected
            value="CP"
    >
      CEO/president/Chairman
    </option>
    <option value="CS">
      Consultant
    </option>
    <option value="DT">
      Diplomat/Sen Govt Officer
    </option>
    <option value="OE">
      Exec Mgmt (EVP/SVP/MD)
    </option>
    <option value="GI">
      Gov&#x27;t/Int&#x27;l org official
    </option>
    <option value="MB">
      MBA Student
    </option>
    <option value="MA">
      Manager/Supervisor
    </option>
    <option value="OT">
      Other
    </option>
    <option value="MD">
      Other C Level (CFO/COO/CIO/CMO)
    </option>
    <option value="OP">
      Owner/Partner/Proprietor
    </option>
    <option value="PO">
      Politician/Government Minister
    </option>
    <option value="PR">
      Professional
    </option>
    <option value="PM">
      Programme/Project Manager
    </option>
    <option value="SE">
      Secretary/Treasurer
    </option>
    <option value="SM">
      Senior Manager/Dept Head
    </option>
    <option value="TS">
      Technical/Business Specialist
    </option>
    <option value="DH">
      VP/Director
    </option>
  </select>
  <div class="o-forms__errortext">
    Please select your position
  </div>
</div>
`;

exports[`Position render a select with a label 1`] = `
<div id="positionField"
     class="o-forms o-forms--wide ncf__field js-field"
     data-ui-item="select"
     data-ui-item-name="position"
     data-validate="required"
>
  <label for="position"
         class="o-forms__label"
  >
    What&#x27;s your job position?
  </label>
  <select id="position"
          name="position"
          class="o-forms__select js-field__input js-item__value"
          data-trackable="field-position"
          aria-required="true"
          required
  >
    <option value>
      Please select a job position
    </option>
    <option value="AN">
      Analyst
    </option>
    <option value="AS">
      Associate
    </option>
    <option value="BR">
      Broker/Trader/Advisor
    </option>
    <option value="BU">
      Business School Academic
    </option>
    <option value="CP">
      CEO/president/Chairman
    </option>
    <option value="CS">
      Consultant
    </option>
    <option value="DT">
      Diplomat/Sen Govt Officer
    </option>
    <option value="OE">
      Exec Mgmt (EVP/SVP/MD)
    </option>
    <option value="GI">
      Gov&#x27;t/Int&#x27;l org official
    </option>
    <option value="MB">
      MBA Student
    </option>
    <option value="MA">
      Manager/Supervisor
    </option>
    <option value="OT">
      Other
    </option>
    <option value="MD">
      Other C Level (CFO/COO/CIO/CMO)
    </option>
    <option value="OP">
      Owner/Partner/Proprietor
    </option>
    <option value="PO">
      Politician/Government Minister
    </option>
    <option value="PR">
      Professional
    </option>
    <option value="PM">
      Programme/Project Manager
    </option>
    <option value="SE">
      Secretary/Treasurer
    </option>
    <option value="SM">
      Senior Manager/Dept Head
    </option>
    <option value="TS">
      Technical/Business Specialist
    </option>
    <option value="DH">
      VP/Director
    </option>
  </select>
  <div class="o-forms__errortext">
    Please select your position
  </div>
</div>
`;

exports[`Position render a select with a label 2`] = `
<div id="positionField"
     class="o-forms o-forms--wide ncf__field js-field"
     data-ui-item="select"
     data-ui-item-name="position"
     data-validate="required"
>
  <label for="position"
         class="o-forms__label"
  >
    What&#x27;s your job position?
  </label>
  <select id="position"
          name="position"
          class="o-forms__select js-field__input js-item__value"
          data-trackable="field-position"
          aria-required="true"
          required
  >
    <option value>
      Please select a job position
    </option>
    <option value="AN">
      Analyst
    </option>
    <option value="AS">
      Associate
    </option>
    <option value="BR">
      Broker/Trader/Advisor
    </option>
    <option value="BU">
      Business School Academic
    </option>
    <option value="CP">
      CEO/president/Chairman
    </option>
    <option value="CS">
      Consultant
    </option>
    <option value="DT">
      Diplomat/Sen Govt Officer
    </option>
    <option value="OE">
      Exec Mgmt (EVP/SVP/MD)
    </option>
    <option value="GI">
      Gov&#x27;t/Int&#x27;l org official
    </option>
    <option value="MB">
      MBA Student
    </option>
    <option value="MA">
      Manager/Supervisor
    </option>
    <option value="OT">
      Other
    </option>
    <option value="MD">
      Other C Level (CFO/COO/CIO/CMO)
    </option>
    <option value="OP">
      Owner/Partner/Proprietor
    </option>
    <option value="PO">
      Politician/Government Minister
    </option>
    <option value="PR">
      Professional
    </option>
    <option value="PM">
      Programme/Project Manager
    </option>
    <option value="SE">
      Secretary/Treasurer
    </option>
    <option value="SM">
      Senior Manager/Dept Head
    </option>
    <option value="TS">
      Technical/Business Specialist
    </option>
    <option value="DH">
      VP/Director
    </option>
  </select>
  <div class="o-forms__errortext">
    Please select your position
  </div>
</div>
`;
